README.md

```diff
@@ -31,7 +31,7 @@ Include Pundit in your application controller:
 ``` ruby
 class ApplicationController < ActionController::Base
   include Pundit
-  protect_from_forgery
+  protect_from_forgery # (Note: you don't need to explicitly add protect_from_forgery for rails 5.2 and above. It will be included in ActionController::Base)
```

Comment:
I think the point here was maybe to demonstrate that Pundit should happen first in the application controller?

Given that rails 4 is past end-of-life I guess we should just remove this line and add a comment saying that Pundit should come first in the controller (though I'm kind of guessing here - I don't know how important that actually is...)

Reply:
I suspect it is out of convenience that pundit is added in the application controller - otherwise it may have to be added in where required. Shall I add in a note `protect_from_forgery` or remove it entirely?